src/model.py

```diff
@@ -2211,25 +2155,33 @@ def forward(self, batch):
                         ],
                         dim=1
                     )
-                else:
+                elif len(self.components[task][f"cnn_{stack}"]) > 0:
+                        
```

Comment:
Previous, we were relying on empty sequential outputs just being input. But now, since `cnn_dim` changes in first cnn block, we needed to handle when some tasks don't use cnns.
(This is the part that could have been implmented differently, but will now be saved for later work)